### PR TITLE
Rename deprecated functions. Also add workaround for nose depracation.

### PR DESCRIPTION
**Pull request by iahepburn**

### CODE REVIEW

File: validation/run_validation_dist_tests.py [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 import os.path as osp
 import sys
+import collections
```
Comment (jlallouette) [inline validation/run_validation_dist_tests.py:5]: I think it works without this line.
```suggestion
```

File: validation/run_validation_mpi_tests.py [hunk 2]
```diff
@@ -2,12 +2,17 @@
 
 import os.path as osp
 import sys
+import collections
```
Comment (jlallouette) [inline validation/run_validation_mpi_tests.py:5]:
```suggestion
```

File: validation/run_validation_tests.py [hunk 3]
```diff
@@ -1,5 +1,8 @@
 import os.path as osp
 import sys
+import collections
```
Comment (jlallouette) [inline validation/run_validation_tests.py:3]:
```suggestion
```

**Review by jlallouette — APPROVED** [review]
LGTM!
One of the imports might not be necessary but it's not very important.

File: validation/run_validation_dist_tests.py (hunk 1 above)
Comment (iahepburn) [reply validation/run_validation_dist_tests.py:5]: OK, yeah, a little superfluous.

**Pull request merged**